cbcontainers/state/common/env_var.go

```diff
@@ -61,6 +65,44 @@ func (b *EnvVarBuilder) WithCustom(customEnvsToAdd ...coreV1.EnvVar) *EnvVarBuil
 	return b
 }
 
+func (b *EnvVarBuilder) WithProxySettings(proxySettings *cbcontainersv1.CBContainersProxySettings) *EnvVarBuilder {
+	if *proxySettings.Enabled == false {
```

Comment:
Potential nil pointer reference. If the value is always set (false/true), then this shouldn't be a pointer, and just be a bool